examples/test_retrieval.metta

```diff
@@ -0,0 +1,3 @@
+!(extend-py! motto)
+!(bind! &retrieval (Agent (retrieval-agent "text_for_retrieval.txt" 200 2 "data")))
+!(llm  &retrieval (user "Can I publish service without using grpc?" ))
```

Comment:
A slightly more complex example would be nice:
* two documents in a subfolder
* the result of the retrieval agent is not returned directly, but inserted as a system message for chat-gpt (e.g., `"(system ("Taking this information into account, answer the user question" (llm &retrieval ...)))`